front/src/modules/simulationResult/components/SpeedSpaceChart/SpeedSpaceChart.tsx

```diff
@@ -237,7 +216,7 @@ export default function SpeedSpaceChart({
    */
   useEffect(() => {
     if (dateIsInRange(timePosition, timeScaleRange)) {
-      traceVerticalLine(chart, CHART_AXES.SPACE_SPEED, positionValues, rotate, timePosition);
+      traceVerticalLine(chart, CHART_AXES.SPACE_SPEED, positionValues, false, timePosition);
```

Comment:
Can you change the order of the parameters of traceVerticalLine to put rotate as the last param and set it by default to false ? Then you can remove it like this. Don't forget to change the param order in the other places where traceVerticalLine is called
```suggestion
      traceVerticalLine(chart, CHART_AXES.SPACE_SPEED, positionValues, timePosition);
```